server/plugins/task-plugin-ob-mysql/src/main/java/com/oceanbase/odc/plugin/task/obmysql/datatransfer/factory/BaseParameterFactory.java

```diff
@@ -76,15 +74,7 @@ public T generate() throws IOException {
         setInitSqls(parameter);
         setFileConfig(parameter, workingDir);
         parameter.setThreads(3);
-        if (transferConfig.getDataTransferFormat() == DataTransferFormat.SQL) {
-            return parameter;
-        }
-        if (transferConfig.getTransferType() == DataTransferType.EXPORT) {
-            setCsvInfo(parameter);
-        } else if (!transferConfig.isCompressed()) {
-            /*
-             * csv 文件导入时需要手动设置 csv 文件相关解析参数
-             */
+        if (transferConfig.getDataTransferFormat() != DataTransferFormat.SQL) {
```

Comment:
This is to avoid a ob-loader-dumper's bug. obloader may fail to parse MANIFEST.bin , so we should set csv config manurally.